validator/testcases/jetpack_data.txt

```diff
@@ -35,7 +35,13 @@
 ./app-extension/bootstrap.js firefox-24-06-06-2013 314b849906e2d9d62cd942f90281f2c4d5fe4385b377ba7b2578185ef9f8d58c
 ./app-extension/bootstrap.js firefox-24-06-12-2013 314b849906e2d9d62cd942f90281f2c4d5fe4385b377ba7b2578185ef9f8d58c
 ./app-extension/bootstrap.js firefox-24-dev 314b849906e2d9d62cd942f90281f2c4d5fe4385b377ba7b2578185ef9f8d58c
-./data/bootstrap.js 0.0.12 63d6b92dc63c78fdb745ec4b2bdf5bc7926b3de90a5058d64b6dbcd7eb72dd26
```

Comment:
It seems strange that this went away but nothing else did. Did that version never actually exist?